Kernel/KBO.cpp

```diff
@@ -549,7 +549,7 @@ void KBO::checkAdmissibility(HandleError handle) const
   for (FunctionSymbol i = 0; i < nFunctions; i++) {
     auto sort = env.signature->getFunction(i)->fnType()->result();
     /* register min function */
-    auto maxFn = maximalFunctions.getOrInit(std::move(sort), [&](){ return i; } );
```

Comment:
`sort` has type `TermList`, and that's trivially-copyable, so moving it is not required and we use it again below.